editor.planx.uk/src/@planx/components/shared/Preview/NavigateToPublishedButton.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+import { InnerContainer } from "./SaveResumeButton";
+
+const NavigateToPublishedButton: React.FC = () => {
+  const testEnvironment = window.location.pathname.endsWith("/draft")
+    ? "/draft"
+    : "/preview";
+
+  const editorLink = window.location.pathname.replace(
+    testEnvironment,
+    "/published",
+  ) as `/${string}`;
+  const redirectLink = `${window.location.origin}${editorLink}`;
```

Comment:
Using this method to define the links feels a bit janky to me, but with the data available, it was the only way I thought about doing it. 

Hoping that the subdomain logic can be handled with the `window.location.origin`